### ISSUE TITLE
Multiple arches start location wrong when set to right side.

### ISSUE THREAD

**Issue by Kimbo3000** [smeighan/xLights#1162]
When I draw in a set of arches from left to right (4 in my case), then set the starting location to be 'blue square' (on the right side).. Each individual arches starting location is on the right, but group of 4 arches as a whole still starts at the left most arch. I checked this by going into the 'States' definition window and hovering my mouse over the individual nodes.

If I try and draw the arches from right to left, they are drawn upside-down. I then flip them over by setting the 'arch tilt' to 180, but when sequencing (and also in the states definition window), they are still all upside-down as when i first draw them.

I found a similar bug report from 2016, which can be found here: https://github.com/smeighan/xLights/issues/454

Thanks
..
Kimbo

**Comment by keithsw1111** [comment]
Not touching this until 3d is out ... I dont want the code clashes.

**Comment by AzGilrock** [comment]
Looking into this now.  I've already got the upside down issue fixed but that is on the 3D branch so it won't be available till we merge branches.  The weird thing about the arches not looking numbered right when blue square is selected is that if you use a SingleLine buffer with an effect moving right it works correctly on screen.  I believe this is what we found on that #454 issue.  It looks good in preview but was wrong on the actual hardware.
